sdk/core/src/shared/getOptionsFromEnvironment.ts

```diff
@@ -78,6 +81,7 @@ function getBasicOptionsFromEnvironment(options: EnvironmentOptions) {
         issuerUrl: options.issuerUrl || DEV_ISSUER_URL,
         registryUrl: options.registryUrl || DEV_REGISTRY_URL,
         verifierUrl: options.verifierUrl || DEV_VERIFIER_URL,
+        bloomVaultUrl: options.vaultUrl || DEV_BLOOM_VAULT_URL,
         vaultUrl: options.vaultUrl || DEV_VAULT_URL,
```

Comment:
This is certainly a bug: `options.vaultUrl` is used here both for bloom vault and for affinidi vault.

We should probably replace `vaultUrl` in `EnvironmentOptions` with `affinidiVaultUrl` and `bloomVaultUrl`.